numba_typing/tests/test_type_annotations.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import type_annotations
+from typing import Union, Dict, List
+
+
+class TestTypeAnnotations(unittest.TestCase):
+
+    def test_get_func_annotations_exception(self):
+
+        def foo(a: int, b, c: str = "string"):
+            pass
+        self.assertRaises(SyntaxError,  type_annotations.get_func_annotations, foo)
```

Comment:
Let's use context manager and get exception message to assert:
```suggestion
        with self.assertRaises(SyntaxError) as raises:
            type_annotations.get_func_annotations(foo)
        self.assertIn('Not found annotation for parameter b', str(raises.exception))
```